src/Util/DataStore.php

```diff
@@ -140,17 +142,17 @@ public function addFile(string $fileUri, string $datasetId, string $type)
         if (preg_match('/^http/', $fileUri)) {
             // Decode any characters escaped in the URL.
             $fileName = urldecode($fileName);
-            $browser = new \Buzz\Browser();
-            $result = $browser->head($fileUri);
-            $status = $result->getHeaders()[0];
-            if (!preg_match('/200/', $status)) {
+            $client = new GuzzleClient();
+            $result = $client->request('HEAD', $fileUri);
```

Comment:
if HEAD method is not supported by the server, what does request return?

Reply:
HEAD is just the header, so anyone should support that.